LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -100,13 +100,18 @@ async Task InitCallBack()
                 if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("IOTEDGE_APIVERSION")))
                 {
                     ioTHubModuleClient = await ModuleClient.CreateFromEnvironmentAsync(settings);
+                    if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("IOTEDGE_TIMEOUT")))
```

Comment:
This is the timeout for getting the twin setting only for the module. Not sure why you would like to have this as settings. We use twin also for devices so if it would be so slow nothing would work.